ldes-server-compaction/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/compaction/CompactionIT.java

```diff
@@ -4,9 +4,9 @@
 import org.junit.platform.suite.api.SelectClasspathResource;
 import org.junit.platform.suite.api.Suite;
 
-@Suite
-@IncludeEngines("cucumber")
-@SelectClasspathResource("features")
-public class CompactionIT extends CompactionIntegrationTest {
-
-}
+//@Suite
+//@IncludeEngines("cucumber")
+//@SelectClasspathResource("features")
+//public class CompactionIT extends CompactionIntegrationTest {
+//
+//}
```

Comment:
uncomment or delete

Reply:
As discussed, this will be disabled until compaction works fine again => TODO added to make this commenting out more clear